src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -96,13 +137,415 @@ void AreaTrigger::Update(uint32 p_time)
         Remove(); // expired
 
     WorldObject::Update(p_time);
+    _timeSinceCreated += p_time;
+
+    if (GetTemplate()->HasFlag(AREATRIGGER_FLAG_HAS_ATTACHED))
+    {
+        if (Unit* target = GetTarget())
+            GetMap()->AreaTriggerRelocation(this, target->GetPositionX(), target->GetPositionY(), target->GetPositionZ(), target->GetOrientation());
```

Comment:
Any ideas about this one? Shauren did a [commit](https://github.com/TrinityCore/TrinityCore/commit/bc758991140e961ceac46b17b766b0d54b8129fb) for fixing spellmods after map change, I thought we could do something similar. Ofcourse, not inside that function, since it's for packets, but before/after calling it? Would the tsrget still be null?